backend/src/main/java/com/nikogrid/backend/dto/RecoveryJwtTokenDTO.java

```diff
@@ -0,0 +1,7 @@
+package com.nikogrid.backend.dto;
+
+public record RecoveryJwtTokenDTO(
+        String token
```

Comment:
You call this `RecoveryJwtTokenDTO` but it doesn't seem to be related to recovery at all. It seems like a normal access token.